settings.gradle

```diff
@@ -77,5 +77,6 @@ include(
     "zgw:zaken-api",
     "zgw:portaaltaak"
 )
-
+include 'zgw:besluiten-api'
+findProject(':zgw:besluiten-api')?.name = 'besluiten-api'
```

Comment:
Not sure why this is needed. Can't it be in the include list above?

Also, can the new besluiten-api dependency be added to the /app/gzac/build.gradle?

Reply:
Missed that, guess it was generated. Fixed it, and also added the besluiten-api dependency to the gzac module